rkt/status.go

```diff
@@ -46,6 +48,7 @@ const (
 func init() {
 	cmdRkt.AddCommand(cmdStatus)
 	cmdStatus.Flags().BoolVar(&flagWait, "wait", false, "toggle waiting for the pod to exit")
```

Comment:
"for the pod to reach a terminal state" is technically more correct; the caller should still ask status afterwards.

I'm not sure if there's a reason not to include a timeout here. I expect to kill the rkt process after a timeout if one isn't provided here regardless.